src/storage/src/sst.rs

```diff
@@ -65,6 +65,7 @@ pub struct LevelMetas {
     levels: LevelMetaVec,
     sst_layer: AccessLayerRef,
     file_purger: FilePurgerRef,
+    compaction_window: Arc<RwLock<Option<i64>>>,
```

Comment:
I think we should use a copy-on-write pattern to update the compaction window like how we update the `LevelMetas`.